src/niswitch/examples/niswitch_connect_channels_grpc.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
+
+import argparse
+import grpc
+import niswitch
+import sys
+
+
+def example(resource_name, channel1, channel2, topology, simulate):
+    # if we are simulating resource name must be blank
+    resource_name = '' if simulate else resource_name
+    # Connect to an already running server using the following server information
+    server_address = "localhost"
```

Comment:
Single quotes everywhere, for consistency with the rest of the file.

Reply:
Updated